### PR TITLE
Minor changes in readme.md

### PR DESCRIPTION
**Pull request by bhav-khurana**
## Description
Made a minor change in the readme.md file

## Checklist:

- [x] I have read the **CONTRIBUTING** guidelines in repository.
- [ ] My pull request has a descriptive title and is attached to an existing issue.
- [x] I haven't repeated the code and it follows established code style of repository.
- [x] My branch is up-to-date with upstream branch.
- [x] All new and existing tests passed.

### CODE REVIEW

File: content/members/members.json [hunk 1]
```diff
@@ -411,5 +411,12 @@
     "github": "https://github.com/PaarthAgarwal",
     "link": "https://www.linkedin.com/in/paarth-agarwal-2291911b8/",
     "year": 2020
+  },
+  {
```
Comment (IamEzio) [inline content/members/members.json:415]: Remove this change. Will add the members later.

**Review by IamEzio — CHANGES_REQUESTED** [review]
Changes in the readme look good.

File: content/members/members.json (hunk 1 above)
Comment (bhav-khurana) [reply content/members/members.json]: Bhaiya, I have removed this change

**Review by IamEzio — APPROVED** [review]
LGTM

**Pull request merged**